brainscore/benchmarks/Lee2020.py

```diff
@@ -0,0 +1,188 @@
+            URL = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116},
+            eprint = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116.full.pdf},
+            journal = {bioRxiv}}'''
+
+
+class DicarloLee2020ITSpatialCorrelation(BenchmarkBase):
+
+    def __init__(self):
+        '''
+        This benchmark compares the distribution of pairwise response correlation as a function of distance between the
+            MajajHong assembly and a candidate assembly
+        Plots of this can be found in Figure 2 of the corresponding publication (see BIBTEX)
+        '''
+        assembly = brainscore.get_assembly('dicarlo.MajajHong2015').sel(region='IT')
+        assembly = make_static(assembly)
+
+        super(self).__init__(identifier='dicarlo.Lee2020.IT-tissue',
+                             ceiling_func=lambda: InternalConsistency()(assembly.transpose('presentation', 'neuroid')),
+                             version=0.1,
+                             parent='IT',
+                             bibtex=BIBTEX)
+
+        self._stimulus_set = assembly.stimulus_set
+        self._target_assembly = average_repetition(assembly)
+        self._target_statistic_fn = self.create_tissue_response_correlation_target
+        self._score = SpatialCorrelationSimilarity(ks_similarity, 1)
+
+        self.bootstrap_samples = 100_000
+        self.num_sample_arrs = 10  # number of simulated Utah arrays sampled from candidate model tissue
+        self._array_size_mm = (np.ptp(self._target_assembly.neuroid.x.data),  # physical size of Utah array in mm
+                               np.ptp(self._target_assembly.neuroid.y.data))
+
+    def __call__(self, candidate):
+        '''
+        :param candidate: NeuroidAssembly
+        :return: Score for the pairwise response correlation compared to the MajajHong Assembly
+        '''
+        candidate.start_recording(recording_target='IT', time_bins=[(70, 170)])
+        activations = candidate.look_at(self._stimulus_set)
+        activations = make_static(activations)
+
+        pairwise_tissue_corr_candidate = []
+        for window in sample_array_locations(activations.neuroid, self.num_sample_arrs, self._array_size_mm):
+            pairwise_tissue_corr_candidate.append(sample_response_corr_vs_dist(activations[window],

```

Comment:
formatting went a bit overboard here, I guess a newline after `append` (and then auto-format again)

Reply:
you might also want to separate this into two lines to make it easier to read, i.e. put the `sample_response_corr_vs_dist` result in a variable that you pass to the list